create-neon/data/templates/README.md.hbs

```diff
@@ -51,6 +51,20 @@ Installs the project, including running `npm run build`.
 
 Builds the Node addon (`index.node`) from source.
 
+Additional [`cargo build`](https://doc.rust-lang.org/cargo/commands/cargo-build.html) arguments may be passed to `npm build` and `npm build-*` commands. For example, to enable a feature flag:
```

Comment:
Since not everyone reading the README will be fully familiar with Rust or Cargo, they may not know that they come with built-in support for feature flags. It might be good to say "to enable a Cargo feature flag". That will give a clue for a Google-able phrase that a new Rust user could use to learn about feature flags in Rust.

Reply:
I thought that was implied by "cargo build arguments" and a link to the cargo docs. You don't think it's clear that `--feature` is applying to `cargo` here?

Reply:
I think because "feature flags" are a generic concept in software engineering, someone who doesn't know that there is build-in support for feature flags in Cargo and Rust might think this was some random example of a bespoke feature flag system. Saying "Cargo feature flags" gives a hint that that's a built-in thing.

Reply:
That seems reasonable. I can add that.